catroid/src/androidTest/java/org/catrobat/catroid/test/utiltests/UtilsTest.java

```diff
@@ -601,4 +602,15 @@ public void testGetBitTooLargeIndex() {
 		assertEquals("Too large index (>=32) should return 0",
 				0, Utils.getBit(0xFFFFFFFF, 32));
 	}
+
+	public void testFormatStringForBubbleBricks() {
+		String testFirstCharWhitespace = " abcdefabcdefabcdefabcdef abcdef abcdefabcdefabcdef";
+		String[] expectedResult = {"abcdefabcdefabcd", "efabcdef abcdef", "abcdefabcdefabcd", "ef"};
```

Comment:
if changed to:
		String testFirstCharWhitespace = " ThisIsAReallyLongishWord toTest TheWordWrapperFunc";
		String[] expectedResult = {"ThisIsAReallyLon", "gishWord toTest", "TheWordWrapperFu", "nc"};

it becomes clear how this function is supposed to work at one glance